app/src/main/res/layout/list_item_album.xml

```diff
@@ -0,0 +1,45 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.list_item_album` appears to be unused
 Unused resources make applications larger and slow down builds.